docs/installation/gateways.md

```diff
@@ -0,0 +1,345 @@
+
+To access the Swarm network in these situations, it may be useful for you to provide a Swarm Gateway, which enables devices with only access to traditional web2 TCP/HTTP connections access to some elements of the swarm.
+
+## Public Gateways
+
+The Swarm Foundation provide access to various public gateways.
+
+### Swarm Foundation Gateway
+
+Use the [Swarm Gateway](https://gateway.ethswarm.org/) to try out Swarm, you are able to upload up to 10mb at a time for free. Upload files, directories, or even folders containing websites with and `index.html` file to host them on Swarm.
+
+You will receive an access link you can then share with others who will be able to retrieve the uploaded file, or browse your website directly from any gateway.
+
+`https://gateway.ethswarm.org/`
+
+### Swarm Foundation Gateway API
+
+If you would like to try out *swarm-cli* or *bee-js* you can start by specfiying the public gateway for operations that do not require access to the *Debug API*.
+
+`https://api.gateway.ethswarm.org/`
+
+### BZZ.link
+
+Use [BZZ.link](https://bzz.link/) to share links using CID's or ENS hosted websites on [Swarm](https://swarm.bzz.link/) in a secure browser context.
+
+`https://bzz.link`
+
+## Limitations
+
+### Rate and Filesize Limits
+
+To ensure everyone is able to try out the Swarm network, rate and filesize limitations are imposed on these public gateways. If you require that yourself, or the users of your Dapp require access which exceeds these limits, you must *run your own gateway*.
+
+### Postage Stamps
+
+When uploading chunks to Swarm, you must provide each with an attached signed postage stamp. Using our public gateway or API, chunks are automatically stamped with random batches that last for about 2 weeks. There is no way to extend or enlarge these batches, or to pick which one you will use.
+
+If you want to be able to control the amount of time your content stays in Swarm, you must have knowledge of your *Batch ID*. In order to choose which chunks are stamped with which batches, you must have access to the relevant key material. Since browser based chunk stamping is not yet implemented in javascript, Gateway Proxy must handle stamping chunks.
+
+## Run Your Own Gateway
+
+Swarm does not yet have a browser based lightnode component, although this feasible and is planned for the future.
+
+In the meantime, in order to access Swarm directly from all your devices, it is possible for you to set use a HTTP gateway. This is similiar to using an RPC provider like *Infura* or *GetBlock* instead of running a blockchain node.
+
+There are three main ways to run a gateway:
+
+### Personal
+
+A personal gateway is to provide Swarm for you and your trusted colleagues, friends and family. This should be protected so that only authenticated users can access it. You can do this by enabling token based authentication, or in the safety of a private network.
```

Comment:
provide an entire swarm network for myself and friends?
this can easily be misunderstood